ros/src/util/packages/runtime_manager/scripts/computing.yaml

```diff
@@ -1917,131 +1924,94 @@ params :
         dash        : ''
         delim       : ':='
 
-  - name  : planner_selector
-    topic : /config/PlannerSelector
-    msg   : ConfigPlannerSelector
-    vars  :
-    - name : latency_num
-      desc : latency_num
-      label: latency_num
-      min  : 0
-      max  : 200
-      v    : 10
-    - name : waypoints_num
-      desc : waypoints_num
-      label: waypoints_num
-      min  : 0
-      max  : 30
-      v    : 8
-    - name : convergence_num
-      desc : convergence_num
-      label: convergence_num
-      min  : 0.0
-      max  : 30.0
-      v    : 2.0
-
-  - name  : decision_maker
+  - name  : decision_maker(experimental)
```

Comment:
This is parameter name, and if you change here, the link will be broken and you can not open it from ```app```.
I think that it is unnecessary to change because it is not a display name.